src/FluxUtilities.cc

```diff
@@ -30,28 +30,28 @@ namespace meas {
 namespace base {
 
 FluxResult::FluxResult()
-        : flux(std::numeric_limits<Flux>::quiet_NaN()),
-          fluxErr(std::numeric_limits<FluxErrElement>::quiet_NaN()) {}
+        : instFlux(std::numeric_limits<InstFlux>::quiet_NaN()),
+          instFluxErr(std::numeric_limits<InstFluxErrElement>::quiet_NaN()) {}
 
 FluxResultKey FluxResultKey::addFields(afw::table::Schema& schema, std::string const& name,
                                        std::string const& doc) {
     FluxResultKey result;
-    result._flux = schema.addField<Flux>(schema.join(name, "flux"), doc, "count");
-    result._fluxErr = schema.addField<FluxErrElement>(schema.join(name, "fluxErr"),
-                                                        "1-sigma flux uncertainty", "count");
+    result._instFlux = schema.addField<InstFlux>(schema.join(name, "instFlux"), doc, "count");
+    result._instFluxErr = schema.addField<InstFluxErrElement>(schema.join(name, "instFluxErr"),
+                                                          "1-sigma instFlux uncertainty", "count");
```

Comment:
Are you sure "Flux" and "FluxErrElement" need renaming? I would have naively expected that the same classes could be used as field types for both kinds of flux.